Update communitytools.js

dmamalis opened the pull request:
Kudzu Analytics platform is coming to Helium. Our advanced RF planner used by LoRaWAN network operators is now available for Helium users. 
Users can plan their installation taking into account the region topography, already installed Helium hotspots and also reward scaling variables. Unlimited simulations are allowed for both free and commercial accounts.
![Peek 2021-07-15 10-43](https://user-images.githubusercontent.com/5211553/126032315-9ba9c9ab-60f2-4248-9008-551f3b3955a4.gif)

dmamalis committed:
Update communitytools.js

danielcolinjames committed:
Update link styling

danielcolinjames committed:
Add 'Planning' category to community tools

danielcolinjames reviewed (approved):
thanks for submitting this @dmamalis! just added a few little tweaks to your PR, and added a new label called "Planning" :)

dmamalis commented:
> thanks for submitting this @dmamalis! just added a few little tweaks to your PR, and added a new label called "Planning" :)

Thanks! Planning is indeed more suitable for now. We will PR soon when the Monitoring Dashboards go live. :-)
